config/locales/en/activemodel.yml

```diff
@@ -978,8 +978,9 @@ en:
               part_employ_evidence_missing: Upload the partner's employment evidence
               court_application_or_order_missing: Upload the application to court or the court order for Section 8 proceedings
               court_application_missing: Upload the application to court for Section 8 proceedings
-              court_order_missing: Upload the court order for Section 8 proceedings
+              court_order_missing: Upload the court order %{matter_type}
               parental_responsibility_missing: Upload the evidence of your client's parental responsibility
+              local_authority_assessment_missing: Upload the local authority assessment
```

Comment:
```suggestion
              local_authority_assessment_missing: Upload the evidence of the local authority's assessment
```

Reply:
I spoke to Quay about this and she suggested the wording that I've got